napari/plugins/pypi.py

```diff
@@ -202,47 +134,12 @@ def iter_napari_plugin_info(
             if name not in skip
         ]
 
-        futures.extend(
-            [
-                executor.submit(
-                    ensure_repo_is_napari_plugin,
-                    (repo_info['full_name'], repo_info['default_branch']),
-                )
-                for repo_info in get_napari_plugin_repos_from_github()
-                if repo_info['name'] not in skip  # repo may not have pkg name
-            ]
-        )
         for future in as_completed(futures):
             info = future.result()
             if info and info not in already_yielded:
                 already_yielded.add(info)
                 yield info
 
 
-@lru_cache(maxsize=1)
-def list_outdated() -> Dict[str, Tuple[str, ...]]:
```

Comment:
It looks like this was being used to find `user_site_packages`, are we sure we don't need it anymore? cc @tlambert03

Reply:
I don't think we need it.  The plugin dialog doesn't use this and has a better way of tagging outdated things.  let's kill it